site/src/pages/WorkspacesPage/WorkspacesPage.tsx

```diff
@@ -54,6 +56,7 @@ const WorkspacesPage: FC = () => {
   const updateWorkspace = useWorkspaceUpdate(queryKey);
   const [checkedWorkspaces, setCheckedWorkspaces] = useState<Workspace[]>([]);
   const [isConfirmingDeleteAll, setIsConfirmingDeleteAll] = useState(false);
+  const [isConfirmingUpdateAll, setIsConfirmingUpdateAll] = useState(false);
```

Comment:
Are there any cases where this could conflict with `isConfirmingDeleteAll`, and the states are mixed together in an invalid way? At least from my quick checks, it seems like only one of these should be true at a time, right?

I'm wondering if there might be a benefit in making them mutually exclusive by turning them into one state value. Something like this:
```tsx
type ActiveUpdate = "deleteAll" | "confirmAll" | "none";
const [activeUpdate, setActiveUpdate] = useState<ActiveUpdate>("none");
```